src/charts/stacked-bar.js

```diff
@@ -473,44 +471,42 @@ define(function(require){
 
         /**
          * Draws the bars along the y axis
-         * @param  {D3Selection} bars Selection of bars
+         * @param  {D3Selection} layersSelection Selection of bars
          * @return {void}
          */
-        function drawVerticalBars(series) {
-            // Enter + Update
-            let bars = series
-                .data(layers)
+        function drawVerticalBars(layersSelection) {
+            let layerJoin = layersSelection
+                .data(layers);
+
+            layerElements = layerJoin
                 .enter()
                   .append('g')
-                    .classed('layer', true)
                     .attr('fill', (({key}) => categoryColorMap[key]))
+                    .classed('layer', true);
```

Comment:
layer seems generic, could this have conflicts with other charts down the road?

Reply:
Mmmm... good call. That could happen yes, I will keep an eye in this.